### PR TITLE
[Dashboard] Support actor detail

### PR DESCRIPTION
**Pull request by rkooo567**
## Why are these changes needed?
Screen Shot 2023-01-31 at 2 46 13 AM

This PR adds actor detail page.

Other than the detail page, it also
- Add pg id to task/actor
- Add profiling links to job detail & job row & actor detail

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

File: dashboard/client/src/pages/job/JobDetail.tsx [hunk 1]
```diff
@@ -114,8 +118,22 @@ export const JobDetailChartsPage = ({
...
+                  <br />
+                  <CpuProfilingLink
+                    pid={job.driver_info?.pid}
+                    ip={job.driver_info?.node_ip_address}
+                  />
+                  <br />
+                  <CpuStackTraceLink
+                    pid={job.driver_info?.pid}
+                    ip={job.driver_info?.node_ip_address}
+                  />
```
Comment (alanwguo) [inline dashboard/client/src/pages/job/JobDetail.tsx:134]: I don't know if it's obvious this refers to profiling for the driver? Should we include text that says Driver cpu profile. or Driver stack trace?

File: dashboard/client/src/App.tsx [hunk 2]
```diff
@@ -229,7 +231,8 @@ const App = () => {
                       />
                     </Route>
                   </Route>
-                  <Route element={<Actors />} path="actors" />
+                  <Route element={<Actors newIA />} path="actors" />
+                  <Route element={<ActorDetailPage />} path="actors/:id" />
```
Comment (alanwguo) [inline dashboard/client/src/App.tsx:235]: This should be nested under Actors similar to how the Jobs and Clusters page works

File: dashboard/client/src/pages/job/JobDetail.tsx (hunk 1 above)
Comment (scottsun94) [reply dashboard/client/src/pages/job/JobDetail.tsx:134]: Good point. I thought it was for this actor..

Comment (rkooo567) [reply dashboard/client/src/pages/job/JobDetail.tsx:134]: Hmm why isn't it obvious though? It is in the job row?

**Review by alanwguo — APPROVED** [review]
approving assuming feedback is addressed

**Pull request merged**